frontend/components/unauthorised-access.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from "./ui/button";
+
+export default function UnauthorisedAccess() {
+  return (
+    <div className="flex items-start justify-center h-screen">
+      <div className="text-center mt-[20vh]">
+        <h1 className="text-4xl font-extrabold tracking-tight lg:text-5xl mb-6">
+          Uh Oh! You're not supposed to be here!
+        </h1>
+        <Button
+          size="lg"
+          onClick={() => {
+            window.location.href = "/";
```

Comment:
Are we going to redirect back to the main landing page for all instances of unauthorised access? I would assume that we will redirect to the user dashboard if user is logged in or redirect to the landing page if user is not logged in.

Reply:
For now the redirect is a placeholder since we do not have a home page up, maybe we can pass in a parameter checking if the user is logged in and redirect based on that?